frontend/packages/console-app/src/redux/actions/cloud-shell-actions.ts

```diff
@@ -1,13 +1,31 @@
+import { useCallback } from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useDispatch, Dispatch } from 'react-redux';
 import { RootState } from '@console/internal/redux';
-import { Dispatch } from 'react-redux';
 import { action, ActionType } from 'typesafe-actions';
 import { isCloudShellExpanded, isCloudShellActive } from '../reducers/cloud-shell-selectors';
 
 export enum Actions {
   SetCloudShellExpanded = 'setCloudShellExpanded',
   SetCloudShellActive = 'setCloudShellActive',
+  SetCloudShellCommand = 'SetCloudShellCommand',
```

Comment:
Just to be consistent with all the other actions.

```suggestion
  SetCloudShellCommand = 'setCloudShellCommand',
```

Reply:
@sahil143 ^^